pytest_metadata/plugin.py

```diff
@@ -85,7 +91,9 @@ def pytest_configure(config):
     }
     config._metadata.update({k: v for k, v in config.getoption("metadata")})
     config._metadata.update(json.loads(config.getoption("metadata_from_json")))
-
+    if config.getoption("metadata_from_json_file"):
+        with open(config.getoption("metadata_from_json_file"), "r") as json_file:
+            config._metadata.update(json.loads(json_file.read()))
```

Comment:
You should be able to do:

```python
config._metadata.update(json.load(json_file))
```

Reply:
Indeed. This should be more readable and faster. Fixed.